### ISSUE TITLE
how to translate image like cyclegan？

### ISSUE THREAD

**Issue by shenhaiyoualn** [suxuann/ddib#6]
I noticed you didn't post the code about image translation like cyclegan, did you？

**Comment by suxuann** [comment]
Nope, I haven't uploaded the code for translating images. I will do that after the Christmas vacation. Stay tuned!

**Comment by miquel-espinosa** [comment]
Any chance on this code being uploaded? 
Also, it would be appreciated if the code for the color conversion with optimal transport (section E.1 in the paper), was also provided. 
Thanks!
